local/compose/labels.go

```diff
@@ -19,24 +19,26 @@ package compose
 import (
 	"fmt"
 
-	"github.com/docker/compose-cli/api/compose"
 	"github.com/docker/docker/api/types/filters"
+
+	"github.com/docker/compose-cli/api/compose"
 )
 
 const (
-	containerNumberLabel = "com.docker.compose.container-number"
-	oneoffLabel          = "com.docker.compose.oneoff"
-	slugLabel            = "com.docker.compose.slug"
-	projectLabel         = compose.ProjectTag
-	volumeLabel          = compose.VolumeTag
-	workingDirLabel      = "com.docker.compose.project.working_dir"
-	configFilesLabel     = "com.docker.compose.project.config_files"
-	serviceLabel         = compose.ServiceTag
-	versionLabel         = "com.docker.compose.version"
-	configHashLabel      = "com.docker.compose.config-hash"
-	networkLabel         = compose.NetworkTag
-
-	//ComposeVersion Compose version
+	containerNumberLabel  = "com.docker.compose.container-number"
+	oneoffLabel           = "com.docker.compose.oneoff"
+	slugLabel             = "com.docker.compose.slug"
+	projectLabel          = compose.ProjectTag
+	volumeLabel           = compose.VolumeTag
+	workingDirLabel       = "com.docker.compose.project.working_dir"
+	configFilesLabel      = "com.docker.compose.project.config_files"
+	environmentFilesLabel = "com.docker.compose.project.environment_files"
```

Comment:
docker-compose label is `LABEL_ENVIRONMENT_FILE = 'com.docker.compose.project.environment_file'` (no `s`)